lib/parser/ast/processor.rb

```diff
@@ -39,9 +39,13 @@ def process_variable_node(node)
       def on_vasgn(node)
         name, value_node = *node
 
-        node.updated(nil, [
-          name, process(value_node)
-        ])
+        if value_node
```

Comment:
It can be false? Could you provide an example of such case? I can't get it locally